Tasks/AzureFunctionAppV2/_buildConfigs/Node20/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "vsts-tasks-azurefunctionapp",
+  "version": "1.0.0",
+  "description": "Azure Pipelines Azure Function Deployment",
+  "main": "azurermWebappdeployment.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^20.3.1",
+    "@types/q": "1.0.7",
+    "agent-base": "6.0.2",
+    "azure-devops-node-api": "11.2.0",
+    "azure-pipelines-task-lib": "^4.4.0",
+    "azure-pipelines-tasks-azure-arm-rest": "^3.226.0",
```

Comment:
Suggest we pull AzureFunctions into a seperate PR

Reply:
Removed, will put them into another PR